src/internal/components/configurationPanels/tests/GeoPushpinConfigurationPanel.spec.tsx

```diff
@@ -0,0 +1,43 @@
+// (C) 2020 GoodData Corporation
+import * as React from "react";
+import { shallow } from "enzyme";
+import { IConfigurationPanelContentProps } from "../ConfigurationPanelContent";
+import { DEFAULT_LOCALE } from "../../../../constants/localization";
+import { VisualizationTypes } from "../../../../constants/visualizationTypes";
+import GeoPushpinConfigurationPanel from "../GeoPushpinConfigurationPanel";
+import UnsupportedProperties from "../../configurationControls/UnsupportedProperties";
+
+describe("GeoPushpinConfigurationPanel", () => {
+    function createComponent(props: IConfigurationPanelContentProps) {
+        return shallow<IConfigurationPanelContentProps, null>(<GeoPushpinConfigurationPanel {...props} />, {
+            lifecycleExperimental: true,
+        });
+    }
+
+    const defaultProps: IConfigurationPanelContentProps = {
+        featureFlags: {
+            enablePushpinGeoChart: true,
+        },
+        isError: false,
+        isLoading: false,
+        locale: DEFAULT_LOCALE,
+        type: VisualizationTypes.PUSHPIN,
+    };
+
+    it("should unsupported configuration properties", () => {
+        const wrapper = createComponent({
+            ...defaultProps,
+            references: {
+                a779214c00d24bdb9071c4b2ff6b57ef: "/gdc/md/pid/obj/88",
```

Comment:
`a779214c00d24bdb9071c4b2ff6b57ef ` could be `tooltip_text_reference`, mock data should be simplelized IMO :)

Reply:
updated